### PR TITLE
Added Facilities section to User detail

### PR DESCRIPTION
**Pull request by HejdaJakub**
* In perun admin User detail is now Facilities section.
* Facilities can filtered by id, name or description.

### CODE REVIEW

**Review by bodnara — CHANGES_REQUESTED** [review]
It looks good. There is one small fix needed regarding when "loading" is active.

File: apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  userId: number;
+
+  loading: boolean;
+  filterValue = '';
+  pageSize: number;
+  tableId = TABLE_ADMIN_FACILITIES;
+  displayedColumns = ['id', 'name', 'description'];
+
+
+  ngOnInit(): void {
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts:31]: Added.

File: apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    this.pageSize = this.tableConfigService.getTablePageSize(this.tableId);
+    this.route.parent.params.subscribe(params => this.userId = params['userId']);
+    this.refreshTable();
+  }
+
+  refreshTable() {
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts:40]: Added.

File: apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  filterValue = '';
+  pageSize: number;
+  tableId = TABLE_ADMIN_FACILITIES;
+  displayedColumns = ['id', 'name', 'description'];
+
+
+  ngOnInit(): void {
+    this.pageSize = this.tableConfigService.getTablePageSize(this.tableId);
+    this.route.parent.params.subscribe(params => this.userId = params['userId']);
+    this.refreshTable();
```
[the remark this answers is not in the document]
Comment (HejdaJakub) [reply apps/admin-gui/src/app/users/pages/user-detail-page/user-facilities/user-facilities.component.ts]: Refresh has been moved.

**Review by bodnara — APPROVED** [review]

**Pull request merged**